RELEASE.md

```diff
@@ -0,0 +1,33 @@
+# Open Cluster Management (OCM) Releases
+
+This document provides an overview of the release cadence, versioning strategy, and community involvement for the Open Cluster Management (OCM) project.
+
+## Versioning
+
+OCM follows the [Semantic Versioning Specification](https://semver.org/), which uses a version number format of `MAJOR.MINOR.PATCH`. This approach ensures that version numbers convey meaning about the underlying changes.
+
+## Release Cadence
+
+### Major Releases
+
+TBD
+
+### Minor Releases
+
+The OCM minor releases currently happen approximately four times per year. The minor releases generally covers all the components of OCM to provide developers and users a consistent OCM experience.
+
+### Patch Releases
+
+The OCM patch release is expected to happen approximately a month after a minor release. Not all components are expected to have a minor release. Each component will perform a minor release based on the number of fixes needed from the community.
```

Comment:
I thought that we release patch versions when needed. Will it be released regularly in the future? Please have confirm if it is "Not all components are expected to have a **patch** release?"